BlackCubes/foodtrucks-django-react-native
# Eg 14 fix related name

## Changes
1. Fixed the `related_name` attributes for the `Review` and `Like` models.
2. Initialized the migrations on all models.

## Purpose
The models should have a clear `related_name` attribute for the foreign keys without any clashes.

## Approach
Since there was a `Reverse Accessor Error` on Django when trying to perform a migration, the fix was to change the `related_name` attributes for the `Review` model from "products" to "reviews" on the `product` field, and "users" to "reviews" on the `user` field. This gives a better description on the model with a foreign key (i.e. `product`) where the source was (i.e. `Review` model), or it points to where the `Product` model has a relationship to another model which in turn gets access to that particular model's list. The same thing was done to the `Like` model from "emoji_likes" to "likes" on the `emoji` field, and "product_likes" to "likes" on the `product` field. With all of this done, it fixed the `Reverse Accessor Error` and the migrations were achieved.

## Testing
1. Pull changes.
2. Open the terminal, go to the `/server` folder, type `python manage.py makemigrations --dry-run --verbosity 3` to test the migrations, and make sure it all passes.

Closes #14
